docs/qiita-16S-analysis.rst

```diff
@@ -3,218 +3,424 @@
 
-.. figure::  images/analysis-closed-ref-barchart.png
+.. figure::  images/taxa_barplot.png
    :align:   center
 
-Under "Beta Diversity", you will have a selection of Principle Coordinates
-Analyses of different measures of beta diversity, or the similarity between
-samples.
+The "Taxonomic Level" pull-down allows you to view the taxa within your samples at different specificities. There are 7 level options: 1- Kingdom, 2- Phylum, 3- Class, 4- Order, 5- Genus, 6- Species, 7- Subspecies. 
+
+The "Color Palette" pull-down allows you to change the coloring of your taxa barplot. You can select through “Discrete” palettes in which each taxa is a different color or “Continuous” palettes in which each taxa is a different shade of one color.
+
+The "Sort Sample By" pull-downs allow you to sort your data either by sample metadata or taxonomic abundance and either by ascending or descending order. 
+
+.. _Alpha Diversity Analysis
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Now, let's analyze the alpha diversity of your samples. Alpha diversity metrics describe the diversity of features within a sample or a group of samples. This is used to analyze the diversity within rather than between samples or a group of samples.
+
+One type of analysis for alpha diversity is looking at observed OTUs. This type of analysis will provide the amount of unique OTUs found in a sample or group of samples.
 
-Clicking on one (say, `unweighted unifrac emperor pcoa plot`) will open an
-interactive visualization of the similarity among your samples. Generally
-speaking, the more similar the samples, the closer the are likely to be in
-the PCoA ordination. The Emperor visualization program offers a very useful way
-to explore how patterns of similarity in your data associate with different
-metadata categories. Here, I've colored the points in our test data by cat
-ownership.
+To perform an observed OTU alpha diversity analysis, select the rarefied "dflt_name - BIOM" artifact in the processing network and select "Process". Then select the "dflt_name - BIOM" artifact and select "alpha_diversity" from the drop-down menu. The parameters will appear below the workflow diagram: 
 
-.. figure::  images/analysis-closed-ref-pcoa.png
+.. figure::  images/observed_OTU_parameter.png
    :align:   center
 
-Let's take a few minutes now to explore the various features of Emperor. Open
-a new browser window with the `Emperor tutorial <https://biocore.github.io/emperor/tutorial_index.html#section1>`__ and follow along with your test data.
+Several parameters have been automatically selected for you. In the "p-metric" field we will specify the alpha diversity analysis to run. Enter "observed_otus" in this box, and click "Add Command". Once the command is added the workflow should appear as follows:
```

Comment:
Image is incorrect. It shows the network **after** alpha diversity is completed.